app/src/main/java/org/systers/mentorship/view/fragments/MembersFragment.kt

```diff
@@ -60,6 +64,90 @@ class MembersFragment: BaseFragment() {
 
         (activity as MainActivity).showProgressDialog(getString(R.string.fetching_users))
         membersViewModel.getUsers()
+
+        /**
+         * onClickListener for the FloatingActionButton of Filters.
+         * */
```

Comment:
You can remove this comment as the function clearly states that what it is doing :)